### DISCUSSION TITLE
Unable to login with the API provided

### DISCUSSION THREAD

**Discussion by PandeoF1** [tchiotludo/akhq#1688] [Q&A] [answered]
Hi,

The documentation says "No authentication required" for AKHQ, but I'm consistently getting a "Unauthorized" message when trying to retrieve data through the API.  Is there a way to authenticate to AKHQ using the API?

```bash
$> curl -X GET "http://localhost:8080/api/cluster"  -H "accept: application/json"

{"message":"Unauthorized"}
```

**Comment by PandeoF1** [comment, marked as answer]
Using basic auth in curl works. Is there documentation about that?

```bash
$>  curl -X GET "http://localhost:8080/api/cluster"  -H "accept: application/json" -u xxx:xxx

[{"id":"xxx","registry":false}]
```

**Reply by tchiotludo** [reply]
Seems that you have enabled the auth on AKHQ, so you need to pass the user pass, seems normal

**Reply by PandeoF1** [reply]
Yes but there nothing is the documentation about how to login on the API. (or i didn't find it)

**Reply by tchiotludo** [reply]
what is the auth that you enabled? oidc / basic auth?

**Reply by PandeoF1** [reply]
I wasn't the one on my team who installed AKHQ. However, I noticed the use of basic authentication when I reviewed the configuration file.

I was wondering if there was any documentation on authentication for the API. Because it seems pretty clear to use basic auth when you see the config, but when you look directly from the Doc, you don't understand it right away.
